src/main/java/com/kazuma/quizapplication/QuizJFrame.java

```diff
@@ -132,10 +132,14 @@ private void checkAnswer()
 	    wrongAnswers++;
 	    answerChoiceTextField.setText("Wrong Answer !!");
 	}
+	nextQuestionButton.setEnabled(true);
+	checkAnswerButton.setEnabled(false);
     }
 
     private void loadNextQuestion()
     {
+	checkAnswerButton.setEnabled(true);
+	answerChoiceTextField.setText("");
 	if (questionIndex < questionBank.length)
```

Comment:
Flow should be more like
`if (questionIndex < questionBank.length)`
`	//Do Quiz Question`
`else`
`	//Do finished Quiz work and display results`